packages/Modal/src/components/Content.js

```diff
@@ -0,0 +1,10 @@
+import styled from "styled-components";
+import { spacer } from "@paprika/stylers/lib/helpers";
+
+const Content = styled.div`
+  padding: 0 ${spacer(2)} ${spacer(2)};
```

Comment:
Pretty minor detail, but I'm kinda 😕 on there being bottom padding, but no top padding.  It makes sense this way if the first element is a `<p>` or another `<h>`, since they have margin built in, but I feel like the `Content` should be agnostic.

Reply:
> `Content` should be agnostic of what it contains.

Yes, definitely. But it is not a reason why there is no top padding. There is no top padding, because `Header` already have bottom padding. There will be too much space if we have paddings for both `Header` and `Content`.

But I am thinking about case when there will be only `Content` and no `Header`:

```jsx
<Modal>
  <Modal.Content>
    ...
  </Modal.Content>
</Modal>
```

I will try to implement some solution for this case.

Reply:
@mikrotron Could you please take a look? What do you think about this solution? https://github.com/acl-services/paprika/pull/320/commits/ec0630cb9e835e51863ffd4957d93ca8ce33c4d8

Reply:
I see what you're going for there, but it feels pretty complex.  What about simply using `margin` instead of `padding` for the `Content`, so that the margins will collapse and even if the first child does have some of its own top margin, it won't increase the spacing?

Reply:
Good idea, but:
1. Margins doesn't collapse inside `flex` container.
2. Even if container will not be `flex`, `Content` wrapped with `ContentWrapper`, so margins will not be collapsed anyway.